SVGMatrix: Describe how to create an instance (#10409)
mdn/content: files/en-us/web/api/svgmatrix/index.md

An **`SVGMatrix`** object can be designated as read only, which means that attempts to modify the object will result in an exception being thrown.

> **`SVGMatrix`** does not have a public constructor. You can create a **`SVGMatrix`** through `createSVGMatrix` method on [`SVGSVGElement`](/en-US/docs/Web/API/SVGSVGElement#methods).

Reviewer: ```suggestion
**`SVGMatrix`** does not have a public constructor. You can create an **`SVGMatrix`** using the `createSVGMatrix()` method of [`SVGSVGElement`](/en-US/docs/Web/API/SVGSVGElement#methods).
```
